src/tests/mock_instascan.js

```diff
@@ -0,0 +1,11 @@
+export default {
```

Comment:
Can this also be separated out into a separate PR?

Reply:
See #662